cocotb_build_libs.py

```diff
@@ -386,7 +375,7 @@ def get_ext():
                     os.path.join(share_lib_dir, "fli", "FliCbHdl.cpp"),
                     os.path.join(share_lib_dir, "fli", "FliObjHdl.cpp"),
                 ],
-                extra_link_args=_extra_link_args(rpath="$ORIGIN"),
+                extra_link_args=_extra_link_args(lib_name="libcocotbfli_modelsim", rpath="$ORIGIN"),
```

Comment:
Can we create a `lib_name` variable for consistency here too?

Reply:
Done